deepinv/tests/test_generators.py

```diff
@@ -303,14 +323,778 @@ def test_generation(name, device):
                 ]
             ).to(device)
 
+    elif name == "ProductConvolutionBlurGenerator":
+        w = params["filters"]
+        # atol = 1e-5
+        atol = 1e-2
```

Comment:
Isn't this a bit too high?

Reply:
It passed on my 2 local machines with atol=1e-5 with newest torch version. But it doesn't pass on github.
I'm trying to figure out the problem. It would be great if you can test it on your local machine.

Reply:
It's weird for me. I tested it on multiple local machine and it passed every time. But it never passed on github.

Reply:
@matthieutrs Do you know on which hardware Github runs our tests? I ran the test on all of my local machine, I also tried on Google Colab and it passed every time.

Reply:
All test passed now. Some of PhysicsGenerator requires complex operation like torch.svd, which give different values depending on hardware, torch version, etc, so for the test, I only test on float64 with atol=1e-8.

Reply:
Thanks!